Sources/Bow/Data/ListKW.swift

```diff
@@ -175,117 +175,117 @@ public extension ListKW {
-        return fa.ev().map(f)
+    public func map<A, B>(_ fa: Kind<ForListKW, A>, _ f: @escaping (A) -> B) -> Kind<ForListKW, B> {
+        return fa.fix().map(f)
     }
 }
 
 public class ListKWApplicative : ListKWFunctor, Applicative {
-    public func pure<A>(_ a: A) -> HK<ListKWF, A> {
+    public func pure<A>(_ a: A) -> Kind<ForListKW, A> {
         return ListKW.pure(a)
     }
     
-    public func ap<A, B>(_ fa: HK<ListKWF, A>, _ ff: HK<ListKWF, (A) -> B>) -> HK<ListKWF, B> {
-        return fa.ev().ap(ff.ev())
+    public func ap<A, B>(_ fa: Kind<ForListKW, A>, _ ff: Kind<ForListKW, (A) -> B>) -> Kind<ForListKW, B> {
+        return fa.fix().ap(ff.fix())
     }
 }
 
 public class ListKWMonad : ListKWApplicative, Monad {
-    public func flatMap<A, B>(_ fa: HK<ListKWF, A>, _ f: @escaping (A) -> HK<ListKWF, B>) -> HK<ListKWF, B> {
-        return fa.ev().flatMap({ a in f(a).ev() })
+    public func flatMap<A, B>(_ fa: Kind<ForListKW, A>, _ f: @escaping (A) -> Kind<ForListKW, B>) -> Kind<ForListKW, B> {
+        return fa.fix().flatMap({ a in f(a).fix() })
     }
     
-    public func tailRecM<A, B>(_ a: A, _ f: @escaping (A) -> HK<ListKWF, Either<A, B>>) -> HK<ListKWF, B> {
+    public func tailRecM<A, B>(_ a: A, _ f: @escaping (A) -> Kind<ForListKW, Either<A, B>>) -> Kind<ForListKW, B> {
         return ListKW.tailRecM(a, f)
     }
 }
 
 public class ListKWFoldable : Foldable {
-    public typealias F = ListKWF
+    public typealias F = ForListKW
     
-    public func foldL<A, B>(_ fa: HK<ListKWF, A>, _ b: B, _ f: @escaping (B, A) -> B) -> B {
-        return fa.ev().foldL(b, f)
+    public func foldL<A, B>(_ fa: Kind<ForListKW, A>, _ b: B, _ f: @escaping (B, A) -> B) -> B {
+        return fa.fix().foldL(b, f)
     }
     
-    public func foldR<A, B>(_ fa: HK<ListKWF, A>, _ b: Eval<B>, _ f: @escaping (A, Eval<B>) -> Eval<B>) -> Eval<B> {
-        return fa.ev().foldR(b, f)
+    public func foldR<A, B>(_ fa: Kind<ForListKW, A>, _ b: Eval<B>, _ f: @escaping (A, Eval<B>) -> Eval<B>) -> Eval<B> {
+        return fa.fix().foldR(b, f)
     }
 }
 
 public class ListKWTraverse : ListKWFoldable, Traverse {
-    public func traverse<G, A, B, Appl>(_ fa: HK<ListKWF, A>, _ f: @escaping (A) -> HK<G, B>, _ applicative: Appl) -> HK<G, HK<ListKWF, B>> where G == Appl.F, Appl : Applicative {
-        return fa.ev().traverse(f, applicative)
+    public func traverse<G, A, B, Appl>(_ fa: Kind<ForListKW, A>, _ f: @escaping (A) -> Kind<G, B>, _ applicative: Appl) -> Kind<G, Kind<ForListKW, B>> where G == Appl.F, Appl : Applicative {
+        return fa.fix().traverse(f, applicative)
     }
 }
 
 public class ListKWSemigroupK : SemigroupK {
-    public typealias F = ListKWF
+    public typealias F = ForListKW
     
-    public func combineK<A>(_ x: HK<ListKWF, A>, _ y: HK<ListKWF, A>) -> HK<ListKWF, A> {
-        return x.ev().combineK(y.ev())
+    public func combineK<A>(_ x: Kind<ForListKW, A>, _ y: Kind<ForListKW, A>) -> Kind<ForListKW, A> {
```

Comment:
Assuming type aliases work in the same way as Kotlin we can replace in the codebase all instances of types such as `Kind<ForListKW, A>` for their abbreviated aliased form `ListKOf<A>` which is much more readable.